osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-dsmr22.json

```diff
@@ -989,6 +989,90 @@
           "access": "R"
         }
       ]
+    },
+    {
+      "tag": "ACTIVITY_CALENDAR",
+      "description": "Activity calendar",
+      "note": null,
+      "class-id": 20,
+      "version": 0,
+      "obis": "0.0.13.0.0.255",
+      "group": "ABSTRACT",
+      "meterTypes": ["SP","PP"],
+      "attributes": [
+        {
+          "id": 2,
+          "description": "calendar_name_active",
+          "datatype": "octet-string",
+          "valuetype": "DYNAMIC",
```

Comment:
For type DYNAMIC, the field `value` can be removed.

Reply:
removed the value part